vrtool_docs/Preprocessing/Vakindeling.md

```diff
@@ -18,56 +18,34 @@ Dit bestand heeft de volgende kolommen die ingevul dmoeten worden:
+| stabiliteit    	  | Optioneel 	| Moet overeen komen  met een waarde in de kolom `doorsnede` bij [Stabiliteit_default.csv](Macro.md). Hiermee wordt aangegeven welke doorsnede bij welk vak hoort. Let op: bij het maken van de vakindeling is deze invoer optioneel, maar bij het maken van een [database](Genereren_database.md) moeten alle doorsneden in het `geojson` bestand zijn ingevuld.   
+| stabiliteit    	  | Optioneel 	| Moet overeen komen  met een waarde in de kolom `doorsnede` bij [Stabiliteit_default.csv](Macro.md). Hiermee wordt aangegeven welke doorsnede bij welk vak hoort. Let op: bij het maken van de vakindeling is deze invoer optioneel, maar bij het maken van een [database](Genereren_database.md) moeten alle doorsneden in het `geojson` bestand zijn ingevuld.     	                     |
 | kunstwerken 	     | Optioneel 	| *Optie om Kunstwerken te berekenen nog niet beschrikbaar*                                                                                                                                                                                                                                                                                                                     |
-| opmerkingen 	     | Optioneel 	| extra opmerkingen, moet altijd de laatste kolom zijn                                                                                                                                         	                                                                                                                                                                                |
+| opmerkingen 	     | Optioneel 	| extra opmerkingen, moet altijd de laatste kolom zijn.                                                                                                                                                         |
 
-Voorbeeld van hoe kolom 'overslag' ingevuld moet worden:
-![](Filling_Vakindeling_overslag.PNG)
+In onderstaande figuur is met een voorbeeld voor stabiliteit geillustreerd hoe de koppeling tussen doorsnedes en de vakindeling moet worden ingevoerd. Merk op dat het mogelijk is voor meerdere vakken dezelfde doorsnede te hanteren (een voorbeeld in de figuur is de dikgedrukte doorsnede `ET_VOLDOET`). Deze hoeft dan slechts 1x genoemd te worden in het STBI invoerbestand, maar kan bij meerdere vakken worden gebruikt.
+![](Filling_Vakindeling_stbi.PNG)
 
-Belangrijk bij het genereren van de vakindeling zijn met name de `m_start` en `m_eind` parameters. 
+Belangrijk bij het genereren van de vakindeling zijn met name de `m_start` en `m_eind` parameters. Wanneer de lengte van het traject teveel afwijkt van de lengte van de shape uit het Nationaal Basisbestand Primaire Waterkeringen wordt een foutmelding gegeven.
 
 **Let op:** 
```

Comment:
Je zou ook hier nog eens kunnen verwijzen naar die totale lengte in de csv en de geojson en dat die overeen moeten komen.